src/config/kube.rs

```diff
@@ -336,6 +341,9 @@ impl Config {
                         KlusterAuth::with_userpass(username, password)
                     )
                 }
+                &UserAuth::Exec(ref _exec) => {
```

Comment:
nit: no `_` at start of var here (that's only for variables you don't reference and don't want linting on)

Reply:
Done